config/routes.rb

```diff
@@ -19,6 +19,10 @@
     concerns :range_searchable
   end
 
+  resource :bento, controller: :bento, defaults: { format: :json }, only: [:index], as: 'bento', path: '/bento' do
```

Comment:
specify `controller` so it can be singular and not `BentosController`

Reply:
makes sense!